frontend/app/components/organisms/ViewMeeting/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import styles from '../../../../styles/ViewMeeting.module.scss';
+
+type ViewMeetingDetailsProps = {
```

Comment:
For these props, I would recommend trying to match the props that are included in the actual meeting model:
```
model Meeting {
  id             String   @id @default(auto()) @map("_id") @db.ObjectId
  mid            String   @unique
  title          String
  description    String
  creator        String   
  group          String   
  startDateTime  DateTime
  endDateTime    DateTime
  zoomAccount    String?   
  zoomLink       String?
  zid            String?
  type           String
  room           String
}
```
For instance, I would get rid of the pandaDocs-related props for now since this functionality still needs to be added. You can keep the recurrence prop, but make sure it's kept as optional.